src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/documents/DocumentApiClient.kt

```diff
@@ -12,6 +12,21 @@ import java.util.UUID
 
 @Service
 class DocumentApiClient(@Qualifier("oauthDocumentApiClient") val documentApiClient: WebClient) {
+
+  // Swagger documentation: https://document-api-dev.hmpps.service.justice.gov.uk/swagger-ui/index.html#/document-controller/downloadDocumentFile
+  fun downloadDocumentFile(documentUuid: UUID): ByteArray = documentApiClient.get()
```

Comment:
/*
 This is a multi-line comment.
 It spans multiple lines.
*/
May be add comments like this